libs/expo/betterangels/src/lib/screens/Client/index.tsx

```diff
@@ -67,21 +67,14 @@ export default function Client({
         </Pressable>
       ),
       headerRight: () => (
-        <Pressable
-          accessibilityRole="button"
-          accessible
-          accessibilityHint="goes to previous screen"
+        <TextButton
+          regular
+          color={Colors.WHITE}
+          fontSize="md"
+          accessibilityHint="goes to Edit screen"
+          title="Edit"
           onPress={() => router.navigate(`/edit-client/${id}`)}
-        >
-          <TextButton
-            regular
-            color={Colors.WHITE}
-            fontSize="md"
-            accessibilityHint="goes to Edit screen"
-            title="Edit"
-            onPress={() => router.navigate(`/edit-client/${id}`)}
-          />
-        </Pressable>
+        />
```

Comment:
same comment here as https://github.com/BetterAngelsLA/monorepo/pull/492#discussion_r1699091758. I assume this change got committed or merged in accidentally at some point?

Reply:
i guess. so changed to fix